torch/fx/experimental/merge_matmul.py

```diff
@@ -29,7 +29,7 @@ def split_result_tensors(result: torch.Tensor, inputs: List[torch.Tensor]) -> Li
     else:
         splits = [x.shape[0] for x in inputs]
 
-    return torch.split(result, splits)
+    return list(torch.split(result, splits))
```

Comment:
I am not sure if it is better to update the returned value to match the type hint or update the type hint to match the returned value.

Reply:
As it's in experimental folder, I think updating the value is fine...

Reply:
There is no type hint errors, but it looks like this change breaks some tests. I will try to update the type hint to match the returned value instead of updating the returned value to match the type hint.

Reply:
All the tests are passing after this change. The second option was the good one